_layouts/docs.html

```diff
@@ -185,6 +185,7 @@
             </div>
             <div {% if page.notoc %} class="col-xs-12 col-sm-9 col-md-10" {% else %} class="col-xs-12 col-sm-9 col-md-8 col-xl-9" {% endif %} >
               <section class="section" id="DocumentationText">
+								<h1>{{ page.title }}</h1>
```

Comment:
Wrapped in an "if" so that this only happens if there is a title. Which should be always, but yes, that will make it so we default to first header (which happens just through natural content rendering)